src/ios/RNCSlider.m

```diff
@@ -10,6 +10,22 @@
 @implementation RNCSlider
 {
   float _unclippedValue;
+  UITapGestureRecognizer * tapGesturer;
+}
+
+- (instancetype)initWithFrame:(CGRect)frame
+{
+    self = [super initWithFrame:frame];
+    if (self) {
+        tapGesturer = [[UITapGestureRecognizer alloc] initWithTarget: self action:@selector(tapHandler:)];
+        [tapGesturer setNumberOfTapsRequired: 1];
+        [self addGestureRecognizer:tapGesturer];
+    }
+    return self;
+}
+- (void)tapHandler:(UITapGestureRecognizer *)gesture {
+    CGPoint touchPoint = [gesture locationInView:self];
+    [self setValue:touchPoint.x / self.bounds.size.width animated: YES];
```

Comment:
@shwetsolanki  / @michalchudziak  this was so close to correct - I wanted tap to move the slider on android and ios so I dug in here because it was reverted in #136 - it looks to me like this is generating the "percent of the slider that was tapped on" but the manager method that handles the value-set is expecting "percent of slider scaled up to maximumValue" and indeed this works when I test it:

```objective-c
    [self setValue:self.maximumValue * touchPoint.x / self.bounds.size.width animated: YES];
```

I'll post a fresh PR that implements it this way with this commit picked as the start of it